docker/Dockerfile-Jenkins-Build-Env

```diff
@@ -4,7 +4,7 @@ FROM ubuntu:18.04
 
 # Install all the pre-requisits we need in the docker image
 RUN apt-get update \
-  && apt-get install -y qemu-user-static docker.io sudo bzip2
+  && apt-get install -y qemu-user-static docker.io sudo bzip2 ccache
```

Comment:
Isn't ccache useless here? We never run a compiler in this container

Reply:
with `docker build`: `RUN make -j8` the local gcc should be used. But using ccache is still not configured within jenkins. There are still more works to do. It is a preparation.

Reply:
Is ccache now required here or not? Its license is not compliant with EPL.

Reply:
Hi. 
It is just our build environment, it has nothing to do with the software  (and it is a complete ubuntu 18.04, we can not clear everything in there)

Reply:
`ccache` removed